src/styles/_header.scss

```diff
@@ -0,0 +1,82 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.img {
```

Comment:
The class name '.img' is too generic and might conflict with other styles. Consider using a more specific class name to avoid potential conflicts.